crates/cast/src/starknet_commands/account/mod.rs

```diff
@@ -57,11 +57,13 @@ pub fn prepare_account_json(
     account_json
 }
 
+#[allow(clippy::too_many_arguments)]
 pub fn write_account_to_accounts_file(
     path_to_scarb_toml: &Option<Utf8PathBuf>,
     rpc_url: &str,
     account: &str,
     accounts_file: &Utf8PathBuf,
+    keystore: &Utf8PathBuf,
```

Comment:
sorry, I'm a bit confused - up until now I was under the impression we use separate files for starkli accounts, and that is the only scenario we want to support. If that's true, why we need keystore in accounts file?

Reply:
My mistake, it's not needed here.